calculator.js

```diff
@@ -0,0 +1,47 @@
+const operators = ['+', '-', '*', '/'];
+
+exports.create_calculator = function (operator) {
+  const numbers = [];
+
+  return {
+    add_number: function (number) {
+      numbers.push(number);
+    },
+    calculate: function () {
+      // Return zero if there are no numbers to calculate
+      if (numbers.length === 0) {
+        throw 'Cannot calculate a result because no numbers were given';
+      }
+
+      let answer = numbers[0];
+
+      let i = 0;
+      numbers.forEach(function (number) {
+        if (i === 0) {
+          i++;
+          return;
+        }
+
+        switch (operator) {
+          case '+':
+            answer = answer + number;
+            break;
+          case '-':
+            answer = answer - number;
+            break;
+          case '/':
+            answer = answer / number;
+            break;
+          default:
+            console.log("Invalid operator " + operator);
+        }
+        // console.log('index: ' + index);
+        // console.log('current answer: ' + answer);
+
+        i++;
+      });
+
```

Comment:
There's really three different things you are pointing out in this comment. 

1: Multiplication is missing
2: the Loop can be improved
3: comments should be removed

It's better to submit these as separate review comments, because when the developer is working through making these improvements, it makes their life easier if they can tick off review comments one by one, as they resolve the problems one by one.